libs/wingc/src/jsify.rs

```diff
@@ -34,8 +34,10 @@ function __app(target) {
 	}
 }
 const $App = __app(process.env.WING_TARGET);
+const $Plugins = plugins;
```

Comment:
What's the use of this global? Can we avoid it?

Reply:
I could pass the plugins as an env var string (then split it in preflight.js). Curious, because I may be missing it, what the downside of leaving as is would be. Just name collisions?